rmgpy/molecule/draw.py

```diff
@@ -1583,8 +1610,8 @@ def draw(self, reaction, format, path=None):
         from .molecule import Molecule
         from rmgpy.species import Species
```

Comment:
Should these imports be in the main import block? Also, get rid of the relative import